plonk-core/src/proof_system/preprocess.rs

```diff
@@ -214,14 +224,18 @@ where
 
     /// The verifier only requires the commitments in order to verify a
     /// [`Proof`](super::Proof) We can therefore speed up preprocessing for the
-    /// verifier by skipping the FFTs needed to compute the 8n evaluations.
-    pub fn preprocess_verifier(
+    /// verifier by skipping the FFTs needed to compute the 4n evaluations.
+    pub fn preprocess_verifier<PC>(
         &mut self,
-        commit_key: &Powers<E>,
-        transcript: &mut TranscriptWrapper<E>,
-    ) -> Result<widget::VerifierKey<E, P>, Error> {
+        commit_key: &PC::CommitterKey,
+        transcript: &mut Transcript,
+        _pc: PhantomData<PC>,
+    ) -> Result<widget::VerifierKey<F, PC>, Error>
+    where
+        PC: HomomorphicCommitment<F>,
+    {
         let (verifier_key, _, _) =
-            self.preprocess_shared(commit_key, transcript)?;
+            self.preprocess_shared(commit_key, transcript, _pc).unwrap();
```

Comment:
Why do we unwrap and then wrap the result into a `Result` again?

Why not just doing:
```suggestion
            self.preprocess_shared(commit_key, transcript, _pc)
```

Reply:
ah, leftover from when `?` didn't work temporarily